config/packages/oneup_flysystem.yaml

```diff
@@ -31,6 +28,3 @@ oneup_flysystem:
 
         rbdigital_ftp:
             adapter: rbdigital_ftp
-
-        bogportalen_local_zip:
-            adapter: bogportalen_zip
```

Comment:
Not sure why removal of this is related to a new importer?

Reply:
Wasn't being used. And yes - technically clean up should happen in a separate PR.